client/debug.log

```diff
@@ -0,0 +1 @@
+[0126/133255.579:ERROR:directory_reader_win.cc(43)] FindFirstFile: Sistem belirtilen yolu bulamıyor. (0x3)
```

Comment:
Probably should be deleted.

Reply:
indeed

Reply:
I can't delete debug.log will just push it clear?